grails-app/domain/org/pih/warehouse/inventory/CycleCount.groovy

```diff
@@ -94,4 +94,19 @@ class CycleCount {
         }
         return null
     }
+
+    /**
+     * @return The largest count index of all the cycle count items. Helps determine what count we're on.
+     */
+    Integer getMaxCountIndex() {
+        return cycleCountItems.max{ it.countIndex }?.countIndex
```

Comment:
I don't think this is possible (maybe if cycleCountItems is an empty list?), but if this could return a null, we might want to consider Elvis operator'ing a zero at the end.